profiles/default.yml

```diff
@@ -13,6 +13,12 @@ charts:
         - key: vector
           label: bootstrapped
 
+    - name: Vector -from-tasty
+      url: https://github.com/liufengyun/bench/blob/master/tests/Vector.scala
+      lines:
+        - key: vector-from-tasty
+          label: bootstrapped
+
```

Comment:
If you want to show them in the same chart, just add a new line under `name: Vector`